assets/strings/spa.json

```diff
@@ -64,5 +64,7 @@
     "Contact": "Contactar",
     "Documentation": "Documentación",
     "About_Apertium": "Acerca de Apertium",
-    "Apertium_Downloads": "Descargas de Apertium"
+    "Apertium_Downloads": "Descargas de Apertium",
+    "Unobtrusive_Title": "Inconveniencia",
+    "Unobtrusive_Warning": "Somos altamente tristes para la inconveniencia causada pero nuestro servidor aparece cargado ahora. Para poner apertium localmente, complacer referir a esto <a href='http://wiki.apertium.org/wiki/Apertium-apy' target='_blank'> enlace</a>."
```

Comment:
> Do you speak Spanish at a ~native level?

Nope, this is for demo purposes only! I would remove this. Would keep only english.